### DISCUSSION TITLE
Data persistance issue

### DISCUSSION THREAD

**Discussion by sottey** [stringer-rss/stringer#1106] [Q&A] [answered]
I have installed Stringer as a docker container and everything is amazing, but if I restart the docker instance, everything is as if I had not set it up (I lose feeds, history and have to set up the password again)

Here is my docker command:
`sudo docker run -d -e DATABASE_URL="sqlite3:':memory:'" -e POSTGRES_PASSWORD=REDATED -e POSTGRES_USER=REDACTED -e POSTGRES_DB=REDACTED -v ~/docker-data/stringer-data:/var/lib/postgresql/data --name feed -p 8087:8080 mdswanson/stringer`

Any ideas? Thanks!
sottey

**Comment by mockdeep** [comment, marked as answer]
Hey @sottey, I may not be understanding, but setting `DATABASE_URL="sqlite3:':memory:'"` looks suspicious to me. Using an in-memory (non-persisted) db inside the Docker container is going to cause it to evaporate after the machine shuts down. SQLite might work, but we don't support it, so you're best off using Postgres.

**Reply by sottey** [reply]
Thank you for your reply. I am sure that is it.
